inc/configsecurity.class.php

```diff
@@ -252,7 +252,7 @@ function getSNMPAuthProtocol($id) {
     * @param string $p_value
     */
    function showDropdownSNMPEncryption($p_value=NULL) {
-      $encryptions = array(0 => '-----', 'DES', 'AES128', 'AES192', 'AES256', 'Triple-DES');
+      $encryptions = array(0 => '-----', 'DES', 'AES128', 'Triple-DES');
```

Comment:
Can you please replace array() by [] ?
Please also replace '-----' by Dropdown::EMPTY_VALUE